src/helpers/validator.js

```diff
@@ -54,6 +54,30 @@ const isValidPassword = () => check('password').isLength({ min: 8 })
   .isAlphanumeric()
   .withMessage('password should contain only numbers and alphabets');
 
+/**
+   * @param {String} field
+   * @returns {Object} - Express-validator
+   */
+const isNotEmpty = field => check(field)
+  .trim()
+  .not()
+  .isEmpty()
+  .exists()
+  .withMessage(`${field} is a required field`);
+
+/**
+   * @param {String} field
+   * @returns {Object} - Express-validator
+   */
+const isValidGenre = field => check(field)
+  .trim()
+  .not()
+  .isEmpty()
+  .exists()
+  .withMessage(`${field} is a required field`)
+  .isIn(['action', 'thriller', 'romance', 'fiction', 'motivational'])
```

Comment:
I don't think hardcoding this here is the best way to go considering you only covered a few genres